src/store/wallet/wallet-actions.ts

```diff
@@ -15,14 +15,16 @@ export const addHdAccountAction = createAction<AccountInterface>('wallet/ADD-HD-
 export const loadTokenBalancesActions = createActions<string, TokenBalanceInterface[], string>('assets/LOAD_TOKENS');
 export const loadTezosBalanceActions = createActions<string, string, string>('assets/LOAD_TEZOS');
 
-export const loadTokenSuggestionActions =
-  createActions<Pick<TokenMetadataInterface, 'id' | 'address'>, TokenMetadataInterface, string>(
-    'assets/LOAD_TOKEN_SUGGESTION'
-  );
-export const loadTokenMetadataActions =
-  createActions<Pick<TokenMetadataInterface, 'id' | 'address'>, TokenMetadataInterface, string>(
-    'assets/LOAD_TOKEN_METADATA'
-  );
+export const loadTokenSuggestionActions = createActions<
```

Comment:
I had to regenerate yarn.lock for a lot of times. It looks like one of lint plugins was updated